eZ/Publish/Core/Repository/Permission/PermissionResolver.php

```diff
@@ -62,21 +64,23 @@ class PermissionResolver implements PermissionResolverInterface
      * @param \eZ\Publish\Core\Repository\Helper\RoleDomainMapper $roleDomainMapper
      * @param \eZ\Publish\Core\Repository\Permission\LimitationService $limitationService
      * @param \eZ\Publish\SPI\Persistence\User\Handler $userHandler
-     * @param \eZ\Publish\API\Repository\Values\User\UserReference $userReference
-     * @param array $policyMap Map of system configured policies, for validation usage.
+     * @param array $policyMap
```

Comment:
`Map of system configured policies, for validation usage` information could be useful